src/components/pages/ProfilePage.tsx

```diff
@@ -27,7 +27,7 @@ export const ProfilePage = () => {
     <Group direction="column" mt={40} spacing={15}>
       <WithTooltip
         tooltipLabel={<Text>This token is used for authentication in your code editor.
```

Comment:
The space should be on this text and not in the anchor. You can do that with `...in your code editor.{" "}`.